app/src/main/java/nl/tudelft/bbw/controller/BlockController.java

```diff
@@ -123,6 +133,14 @@ public final void updateTrustOfBlock(Block block) {
     }
 
     /**
+     * Retrieve the owner of this app
+     * @return the owner of this app
+     */
+    public User getOwnUser() {
```

Comment:
In the activity, you always already have a user, so why would you need this function